pkg/riff/commands/application.go

```diff
@@ -24,7 +24,7 @@ import (
 func NewApplicationCommand(c *cli.Config) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "application",
-		Short:   "<todo>",
+		Short:   "build applications from source using Cloud Foundry buildpacks",
```

Comment:
For consistency, this should be described using a noun phrase. For example, "applications built from source using Cloud Foundry buildpacks".

Reply:
`built` is past tense, which isn't right since this resource is triggering the build